src/components/home/Home.js

```diff
@@ -42,40 +46,47 @@ function Home() {
   const { t } = useTranslation();
   const classes = useStyles();
   const [searchResults, setSearchResults] = useState(null);
+  const [searchInput, setSearchInput] = useState(null);
+  const [range, setRange] = useState(RANGES.ALL.value);
+  const [keywords, setKeywords] = useState(null);
   const { hooks } = useContext(QueryClientContext);
-  const {
-    data: collections,
-    isLoading,
-    isPlaceholderData,
-  } = hooks.usePublicItemsWithTag(PUBLISHED_TAG_ID, {
-    placeholderData: PLACEHOLDER_COLLECTIONS,
-  });
-  const { data: members } = hooks.useMembers(
-    null,
-    isPlaceholderData
-      ? null
-      : [...new Set(collections?.map(({ creator }) => creator).toArray())],
+  const { data: collections, isLoading } = hooks.usePublicItemsWithTag(
+    PUBLISHED_TAG_ID,
+    {
+      placeholderData: PLACEHOLDER_COLLECTIONS,
+      withMemberships: true,
```

Comment:
I removed this property. If you need the memberships you should use `useItemMemberships` (or similar)